tests/PHPUnit/Integration/Tracker/VisitTest.php

```diff
@@ -102,6 +102,45 @@ public function testIsVisitorIpExcluded($excludedIp, $tests)
         }
     }
 
+    public function getExcludeByUrlData()
+    {
+        return array(
+            array(array('http://test.com'), true, array(
+                'http://test.com' => true,
+                'https://test.com' => true,
+                'http://test.com/uri' => true,
+                'http://test.com/?query' => true,
+                'http://xtest.com' => false,
+                'http://x.test.com' => false,
```

Comment:
In the issue description, we discussed that subdomains should be allowed, so stage.example.net and dev.example.net will be tracked when the checkbox is tracked. this is not how this works here?